src/webview/repl-console.ts

```diff
@@ -481,10 +539,15 @@ export class ReplConsole {
             this.readline.repaint();
         },
         "backspace": () => {
-            this.readline.withUndo(() => {
-                paredit.backspace(this.readline);
+            if (this.isKeyMay([ReplPareditKeyMap.STRICT])) {
+                this.readline.withUndo(() => {
+                    paredit.backspace(this.readline);
+                    this.readline.repaint()
+                })
+            } else {
+                this.readline.backspace();
                 this.readline.repaint()
```

Comment:
How come we do not do this `withUndo`? (I tested it and I can undo also for normal mode, so this is probably correct. I am just wondering.)

Reply:
It comes because I forget to do so. No idea why it works. *gg*